condacolab.py

```diff
@@ -118,6 +121,10 @@ def install_from_url(
         Run checks to see if installation was run previously.
         Change to False to ignore checks and always attempt
         to run the installation.
+    restart_kernel
```

Comment:
We need this docs in the rest of the `install_*` functions too. Sorry, just realized!

Reply:
Oops, I missed it as well. Adding it now.